index.html

```diff
@@ -159,58 +159,77 @@ <h3>Leverage the power of open source in the field</h3>
         </p>
-        <p> Streamline your processes and boost productivity with QField's seamless ecosystem. Whether you're in remote
-          locations or managing large-scale projects, our tools have you covered.</p>
       </div>
-      <div class="row">
-        <div class="col-lg-4">
-          <div class="feature">
-            <h2>Work efficiently</h2>
-            <img src="images/point_deploy.svg" alt="Work efficiently">
-            <p>Survey and digitise data in no time. QField is the professional mobile app for QGIS, allowing users to
-              deploy their existing projects to the field.
-            </p>
-          </div>
+    </div>
+    <div class="row">
+      <!-- First row of features -->
+      <div class="col-lg-4">
+        <div class="feature">
+          <img src="images/icons/map-pin2.svg" alt="Smart Maps" style="width: 25%;">
+          <h3>Smart maps</h3>
+          <p style="text-align: center">High precision location, tracking, geofencing, and full QGIS symbology, just like on desktop.</p>
         </div>
-        <div class="col-lg-4">
-          <div class="feature">
-            <h2>Be anywhere</h2>
-            <img src="images/point_sync.svg" alt="Be anywhere">
-            <p>Edit your data on the go. Seamless integration with <a href="https://qfield.cloud">QFieldCloud</a> allows
-              your team to work on your projects anywhere anytime.</p>
-          </div>
+      </div>
+      <div class="col-lg-4">
+        <div class="feature">
+          <img src="images/icons/files.png" alt="Powerful Forms" style="width: 22%;">
+          <h3>Powerful Forms</h3>
+          <p style="text-align: center">Create rich survey forms with constraints, logic, defaults, and validations — all in QGIS.</p>
         </div>
-        <div class="col-lg-4">
-          <div class="feature">
-            <h2>Open anything</h2>
-            <img src="images/point_formats.svg" alt="Open anything">
-            <p>Open a wide range of spatial data formats, connect to industry-leading spatial databases and consume
-              standardised geowebservices.</p>
-          </div>
+      </div>
+      <div class="col-lg-4">
+        <div class="feature">
+          <img src="images/icons/camera1.svg" alt="Capture Everything" style="width: 25%; margin: 14px;">
+          <h3>Capture everything</h3>
+          <p style="text-align: center">Photos, videos, audio, QR code scanning, and NFC — fully integrated.</p>
         </div>
       </div>
     </div>
+    <div class="row">
+      <!-- Second row of features -->
+      <div class="col-lg-4">
+        <div class="feature">
+          <img src="images/icons/synchronize.png" alt="Seamless Sync & Format Support" style="width: 25%">
+          <h3>Seamless sync<br> & format support</h3>
+          <p style="text-align: center">Sync projects and data in real time with QFieldCloud and work with GeoPackages, KML, GPX, georeferenced PDFs, and more.</p>
```

Comment:
We could make 'QFieldCloud' an hyperlink (styled with our beautiful QFC blue) here:

![image](https://github.com/user-attachments/assets/b0aa362e-3e97-42fa-955f-bae2455171b1)